src/rules/order.js

```diff
@@ -225,10 +225,11 @@ function fixOutOfOrder(context, firstNode, secondNode, order, named) {
   const secondRootEnd = findEndOfLineWithComments(sourceCode, secondRoot);
 
   if (firstNode.displayName === secondNode.displayName) {
-    for (const node of [firstNode, secondNode]) {
-      if (node.alias) {
-        node.displayName = `${node.displayName} as ${node.alias}`;
-      }
+    if (firstNode.alias) {
+      firstNode.displayName = `${firstNode.displayName} as ${firstNode.alias}`;
+    }
+    if (secondNode.alias) {
+      secondNode.displayName = `${secondNode.displayName} as ${secondNode.alias}`;
```

Comment:
maybe even, `if (secondNode.alias) { } else if (firstNode.alias) {` since the first node doesn't matter if the second one is set?

Reply:
In my opinion, if I had a code like
```ts
import { Readline as RL, Readline as Interface } from "readline";
```
I would prefer the message `` `Readline as Interface` import should occur before import of `Readline as RL` `` over `` `Readline as Interface` import should occur before import of `Readline` ``

What do you think?

Reply:
ohhhh sorry you're setting each node's displayName separately, i just misread it. this is fine as is :-)